### DISCUSSION TITLE
charge

### DISCUSSION THREAD

**Discussion by zeecode07** [pybamm-team/PyBaMM#3295] [Q&A]
experiment = pybamm.Experiment([
    ("Discharge at 0.1C until 2.5V",
     "Rest for 1 hour",
     "Charge at 0.1C until 4.2V", 
     "Hold at 4.2V until C/50")
] * 2000,
termination="80% capacity")

Is this code a battery charging cycle, starting with discharging with CC and continuing with charging with CCCV? @tinosulzer

**Comment by brosaplanella** [comment]
This is a very basic question on how to use experiments. Please check the documentation first, the [experiment notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/getting_started/tutorial-5-run-experiments.html) will probably answer your questions.
